src/v0/destinations/customerio/util.js

```diff
@@ -236,13 +237,14 @@ const defaultResponseBuilder = (message, evName, userId, evType, destination, me
   // DEVICE registration
   const isDeviceRelatedEvent = isdeviceRelatedEventName(evName, destination);
   if (isDeviceRelatedEvent && userId && token) {
+    const timestamp = message.timestamp || message.originalTimestamp;
     const devProps = {
       ...message.properties,
       id: token,
-      last_used: Math.floor(new Date(message.originalTimestamp).getTime() / 1000),
+      last_used: Math.floor(new Date(timestamp).getTime() / 1000),
     };
     const deviceType = get(message, 'context.device.type');
```

Comment:
what if we convert deviceType to string here itself?

Reply:
Do you mean this `deviceType = '123'` ?